scripts/install-toolchain.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+PLATFORM=$(uname -s | tr '[:upper:]' '[:lower:]')
+ARCH=$([[ $(uname -m) = "x86_64" ]] && echo 'amd64' || echo 'arm64')
+GIT_REPO_ROOT=$(git rev-parse --show-toplevel)
+BUILD_DIR="${GIT_REPO_ROOT}/chartbuild"
+TMP_DIR="${BUILD_DIR}/tmp"
+TOOLS_DIR="${BUILD_DIR}/tools"
+mkdir -p "${TOOLS_DIR}"
+export PATH="${TOOLS_DIR}:${PATH}"
```

Comment:
Those seem to be used by all scripts. Since these being the same is not just incidental but part of the interface, it could make sense to export them from the Makefile instead.